packages/popover/src/vaadin-popover-overlay-mixin.js

```diff
@@ -68,15 +35,6 @@ export const PopoverOverlayMixin = (superClass) =>
         return;
       }
 
-      // Copy custom properties from the owner
```

Comment:
The offset doesn't seem to work at the moment, because these styles override whatever is set on the popover element itself: https://github.com/vaadin/web-components/blob/a49351c8864e4bdac2f2e33fce1b1fa2d0a7bfe7/packages/popover/theme/lumo/vaadin-popover-styles.js#L8-L16

Reply:
Good point, thanks. Looks like `vaadin-tooltip` also has this problem now, I'll create a separate PR for it.